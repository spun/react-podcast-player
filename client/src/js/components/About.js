import React from "react";
import PropTypes from "prop-types";

// Componente que muestra los creditos de la aplicaciÃ³n
const About = ({ list }) => {
	return (
		<div className="card">
			<div className="header">
				<h4 className="title">About this app</h4>
			</div>
			<div className="content">
				<h5>Libraries</h5>
				<ul>
					<li>
						<a
							href="https://facebook.github.io/react/"
							target="_blank">
							React
						</a>
					</li>
					<li>
						<a href="http://redux.js.org/" target="_blank">
							Redux
						</a>
					</li>
					<li>
						<a
							href="https://reacttraining.com/react-router/"
							target="_blank">
							React router
						</a>
					</li>
				</ul>

				<h5>Theme</h5>
				<ul>
					<li>
						<a
							href="https://www.creative-tim.com/product/paper-dashboard"
							target="_blank">
							Paper dashboard
						</a>
					</li>
				</ul>
			</div>
		</div>
	);
};

export default About;
