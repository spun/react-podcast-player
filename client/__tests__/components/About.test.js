import React from "react";
import { MemoryRouter } from "react-router-dom";
import renderer from "react-test-renderer";
// Component under test
import About from "../../src/js/components/About";

test("About section creation", () => {
	const component = renderer.create(<About />);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
