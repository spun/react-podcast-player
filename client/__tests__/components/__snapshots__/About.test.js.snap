// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About section creation 1`] = `
<div
  className="card"
>
  <div
    className="header"
  >
    <h4
      className="title"
    >
      About this app
    </h4>
  </div>
  <div
    className="content"
  >
    <h5>
      Libraries
    </h5>
    <ul>
      <li>
        <a
          href="https://facebook.github.io/react/"
          target="_blank"
        >
          React
        </a>
      </li>
      <li>
        <a
          href="http://redux.js.org/"
          target="_blank"
        >
          Redux
        </a>
      </li>
      <li>
        <a
          href="https://reacttraining.com/react-router/"
          target="_blank"
        >
          React router
        </a>
      </li>
    </ul>
    <h5>
      Theme
    </h5>
    <ul>
      <li>
        <a
          href="https://www.creative-tim.com/product/paper-dashboard"
          target="_blank"
        >
          Paper dashboard
        </a>
      </li>
    </ul>
  </div>
</div>
`;
